packages/code.rb

```diff
@@ -6,11 +6,13 @@ class Code < Package
   version '1.25'
   case ARCH
   when 'x86_64'
+    # code-stable-1534444642.tar.gz
     source_url 'https://go.microsoft.com/fwlink/?LinkID=620884'
```

Comment:
Can we use the GitHub page to get the sources? https://github.com/Microsoft/vscode/releases

That will stop us from downloading from a moving target and hitting checksum errors every time they update.

Reply:
No. These versions are better.